packages/blockchain-wallet-v4-frontend/src/scenes/Dex/Swap/EnterSwapDetails/EnterSwapDetails.tsx

```diff
@@ -49,6 +52,25 @@ export const EnterSwapDetails = ({ walletCurrency }: Props) => {
     selectors.components.dex.getDexCoinBalanceToDisplay(counterToken)
   )
 
+  useEffect(() => {
```

Comment:
Did you try to implement polling logic in sagas instead? E.g., how it is done for `startPollSellQuotePrice` in https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/e9724aaf28cd4743c5879cb4077020e2e461764b/packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts?